package.json

```diff
@@ -116,5 +116,8 @@
         "extension": true,
         "outputDir": "jupyterlite_ai/labextension",
         "schemaDir": "schema"
+    },
+    "resolutions": {
+        "@jupyterlab/fileeditor": "^4.3.5"
```

Comment:
Do we need to keep this resolution?

Reply:
Good catch, thanks.
No, it's not necessary anymore, I used this resolution to avoid too much versions of some jupyterlab and lumino packages in `yarn.lock`.